pkg/apis/minio.min.io/v1/names.go

```diff
@@ -24,12 +24,21 @@ import (
 // MinIOServerName specifies the default container name for Tenant
 const MinIOServerName = "minio"
 
+// MinIODNSInitContainer Init Container for DNS
+const MinIODNSInitContainer = "minio-dns-wait"
+
+// MinIOVolumeInitContainer Init Container for DNS
+const MinIOVolumeInitContainer = "minio-vol-wait"
+
 // KESContainerName specifies the default container name for KES
 const KESContainerName = "kes"
 
 // ConsoleContainerName specifies the default container name for Console
 const ConsoleContainerName = "console"
 
+// InitContainerImage name for init container.
+const InitContainerImage = "alpine:3.12"
```

Comment:
I think we can use `busybox:1.32`  instead of alpine https://hub.docker.com/_/busybox - much tiny image.

Reply:
ok

Reply:
@kerneltime will you be changing this to to busybox?